Automattic/course
# Add Archive Page + pagination pattern on all template

Issue #10 

##  Introduced changes
* Create a pagination pattern to support i18n
* Apply the pattern on Search/Archive and Index templates.


## Testing instructions:
* Go to the archive/search/index page
* Check if the pagination matches the design

<img width="1556" alt="Screen Shot 2022-10-14 at 16 51 49" src="https://user-images.githubusercontent.com/38718/195931262-4868ffa1-782a-4554-9f0d-7b7b3b426089.png">
